plugins/inventory/ntnx_prism_vm_inventory.py

```diff
@@ -126,7 +127,20 @@ def parse(self, inventory, loader, path, cache=True):
             self.validate_certs,
         )
         vm = vms.VM(module)
```

Comment:
Consider a case when offset if not given. Put defaults
```suggestion
        vm = vms.VM(module)
        self.data["offset"]=self.data.get("offset", 0)
```